HACKING

```diff
@@ -171,9 +171,9 @@ Geany requires GTK >= 2.24 and GLib >= 2.28. API symbols from newer
 GTK/GLib versions should be avoided or made optional to keep the source
 code building on older systems.
 
-It is recommended to use the 2.24 API documentation of the GTK 
-libs (including GLib, GDK and Pango) has the advantages 
-that you don't get confused by any newer API additions and you 
+It is recommended to use the 2.24 API documentation of the GTK
+libs (including GLib, GDK and Pango) has the advantages
+that you don't get confused by any newer API additions and you
```

Comment:
I agree in general. In this case it is probably just not worth to make an extra commit for this.